code/modules/mining/mine_turfs.dm

```diff
@@ -279,7 +279,8 @@
 				if(prob(50))
 					M.Stun(5)
 			M.apply_effect(25, IRRADIATE)
-
+			if(prob(25))
```

Comment:
How does this translate to rarity for xenoarch artificacts being excavated with just a pickaxe? As it totally bypasses the xenoarch excavation minigame, it should be pretty rare.